docker-compose.yml

```diff
@@ -1,37 +1,31 @@
 version: '4.2'
 services:
-  auth-service:
-    build:
-      context: ./auth-service
-    depends_on:
-      - auth-service-db
+  auth-service-db:
+    image: postgres:latest
+    environment:
+      POSTGRES_PASSWORD: mysecretpassword
     networks:
       - auth-network
     ports:
-      - "50051:50051"
+      - "5001:5432"
 
```

Comment:
You have lost auth-service in this file, finally the docker compose from main should be taken, it is much more advanced